src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -596,9 +591,12 @@ public final void setSeverity(String severity) {
      * quality of their reports, e.g. to load a class and then analyze it via
      * reflection.
      * @param classLoader the new classloader
+     * @deprecated Checkstyle is not type aware tool and all Checks derived from this
+     *     class are potentially unstable.
```

Comment:
This comment seems to have been copied from another location as is - no Checks should be derived from the class `Checker`.

Reply:
Done